lib/src/transaction/staking/withdraw_unbonded_transaction_builder.ts

```diff
@@ -1,4 +1,5 @@
 import ow from 'ow';
+import lodash from 'lodash';
```

Comment:
Suggest to use `import _cloneDeep from 'lodash/cloneDeep'` to reduce bundle size.